### PR TITLE
Classes

### LINKED ISSUE #6: [1pt] Create Item class
Class OOP

### LINKED ISSUE #11: [1pt] Create a Label class
Class OOP

### LINKED ISSUE #12: [1pt] Create Book Class
Class OOP

### LINKED ISSUE #14: [1pt] Create a Game Class
Class OOP

### LINKED ISSUE #15: [1pt] Create an Author Class
Class OOP

### LINKED ISSUE #17: [1pt] Create genre.rb file
Class OOP

### LINKED ISSUE #31: [1pt] Create music_album.rb file.
Class OOP

### PR DESCRIPTION
**Pull request by eerapheal**
I created all the classes and initialized the Catalog method.

### CODE REVIEW

File: classes/item.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Item
+  def initialize(publish_date, archived, id = Random.rand(10_000))
+    @id = id
+    @publish_date = publish_date
+    @archived = archived
+  end
+end
```
Comment (Doheera-kosi) [inline classes/item.rb:7]: - [ ] Good job done, I like the way you constructed the class

File: classes/games.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class Game < Item
+  def initialize(publish_date, archived, multiplayer, last_played_at, id = Random.rand(10_000))
+    super(publish_date, archived, id)
```
Comment (Doheera-kosi) [inline classes/games.rb:3]: - [ ] Good job implementing the super class. I like how you constructed that.

**Review by Doheera-kosi — APPROVED** [review]
Hi @eerapheal , 



## Highlights
🚀 Linters passed

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Comment by eerapheal** [comment]
Thanks for the comments.

**Pull request merged**